src/sentry/snuba/events.py

```diff
@@ -597,76 +614,87 @@ class Columns(Enum):
         event_name=None,
         transaction_name="transaction_op",
         discover_name="transaction_op",
+        issue_platform_name=None,
         alias="transaction.op",
     )
     TRANSACTION_DURATION = Column(
         group_name=None,
         event_name=None,
         transaction_name="duration",
         discover_name="duration",
+        issue_platform_name="transaction_duration",
```

Comment:
We introduced this column a couple weeks ago and it should be updated here accordingly.